www/templates/default/UNL/VisitorChat/Message/Edit.tpl.php

```diff
@@ -10,6 +10,5 @@
     </fieldset>
     <input type="hidden" name="conversations_id" value='<?php echo $context->conversations_id ?>'/>
     <input type="hidden" name="_class" value='<?php echo get_class($context->getRawObject()); ?>'/>
-
+    <input id='visitorChat_message_submit' class="wdn-button wdn-button-triad" type="submit" value="Submit" name="visitorChat_message_submit" />
```

Comment:
I agree 100% that naming convention is smelly. I'd love to change it. Honestly though, now that we are planning to release these visual updates after we retire the 3.1 version, I think it would be fine to change stuff like this.